android/src/app/organicmaps/util/ThemeSwitcher.java

```diff
@@ -134,6 +135,10 @@ private void changeMapStyle(@NonNull String newTheme, @NonNull String oldTheme)
 
   private void SetMapStyle(@Framework.MapStyle int style)
   {
+    // Because of the distinct behavior in auto theme, Android Auto employs its own mechanism for theme switching.
```

Comment:
Why do we need the distinct behavior for Android Auto? Existing ThemeSwitcher is smart enough and checks lat,lon for getting sunrise/sunset time. This logic is quite useful.